sdk/communication/communication-chat/package.json

```diff
@@ -39,8 +39,7 @@
     "dist-esm/src/",
     "types/",
     "README.md",
-    "LICENSE",
-    "src"
+    "LICENSE"
```

Comment:
@DominikMe can you confirm not shipping source files is ok with you?

Reply:
I restored src and set `@azure/azure-sdk/ts-package-json-files-required` to warn instead (similar to other changes where we couldn't confirmation from package owners)